Gems/Multiplayer/Code/Source/MultiplayerStatSystemComponent.cpp

```diff
@@ -209,6 +209,25 @@ namespace Multiplayer
         AZLOG_WARN("Stat with id %d has not been declared using DECLARE_PERFORMANCE_STAT", uniqueStatId);
     }
 
+    void MultiplayerStatSystemComponent::IncrementStat(int uniqueStatId)
+    {
+        AZStd::lock_guard lock(m_access);
+        const auto statIterator = m_statIdToGroupId.find(uniqueStatId);
+        if (statIterator != m_statIdToGroupId.end())
+        {
+            if (const auto group = m_statGroups.Find(statIterator->second))
+            {
+                if (CumulativeAverage* stat = group->m_stats.Find(uniqueStatId))
```

Comment:
Do you need to guard against nullptr here?

Reply:
Yep, it's a null check already due to being inside `if ()`.